composer.json

```diff
@@ -7,7 +7,8 @@
         "php": ">=5.4.0",
         "evenement/evenement": "~2.0",
         "react/event-loop": "0.4.*",
-        "react/stream": "0.4.*"
+        "react/stream": "0.4.*",
+		"phpunit/phpunit": "~4.0"
```

Comment:
Same comment as https://github.com/reactphp/child-process/pull/10/files#r25787513. Not necessary, IMO.